assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -189,6 +191,7 @@ function DashboardSummaryWidget( { Widget, WidgetReportZero, WidgetReportError }
 						className="overview-adsense-impressions"
 						title={ __( 'Ad Impressions', 'google-site-kit' ) }
 						datapoint={ period.totals[ 2 ] }
+						change={ period.totals[ 2 ] - previousPeriod.totals[ 2 ] }
```

Comment:
I think we also need a `changeDataUnit` here, otherwise this looks inconsistent with other numbers that are formatted: 
![image](https://user-images.githubusercontent.com/1621608/116586470-2d5d3f00-a922-11eb-9cf5-daa473590abe.png)

Unfortunately, we don't have a shorthand option for `DataBlock` to use the default "metric" style (which is what we'd want here I think). Passing an empty object would work but that would also be a bit too cryptic for my taste.

I think the easiest thing we can do is pass `changeDataUnit` as a boolean but leave it empty to use the default (`true`) which will also result in the default format, which I think makes more sense than adding something else to mean "use the default".
```suggestion
						change={ period.totals[ 2 ] - previousPeriod.totals[ 2 ] }
						changeDataUnit
```
How does that sound @felixarntz ?

Reply:
@aaemnnosttv Sounds good to me!